show/main.py

```diff
@@ -89,7 +89,7 @@
 # bash oneliner. To be revisited once routing-stack info is tracked somewhere.
 def get_routing_stack():
     result = None
-    command = "sudo docker ps | grep bgp | awk '{print$2}' | cut -d'-' -f3 | cut -d':' -f1 | head -n 1"
+    command = "sudo docker ps | grep bgp | grep -E 'quagga|frr' | awk '{print$2}' | cut -d'-' -f3 | cut -d':' -f1 | head -n 1"
```

Comment:
Do we even need to keep support for quagga? Current SONiC versions can only be built with FRR.

Reply:
Hi, @saiarcot895 , Could you please help to review the PR again when free, if no more comment, I plan to merge it tomorrow, thanks a lot.